engine/src/atmospheric_fog_mesh.cpp

```diff
@@ -0,0 +1,17 @@
+#include "atmospheric_fog_mesh.h"
```

Comment:
Don't forget to add the copyleft notice at the top of this file. And all the others as applicable.